### ISSUE TITLE
Dijkstra path construction fails with error when it reaches node where it cannot continue

### ISSUE THREAD

**Issue by gecoombs** [antarctica/PolarRoute#211]
While working on the glider paths I discovered an example where increasing the size of the mesh causes a previously valid dijsktra path to no longer be found. Running `rp.compute_routes()` gives the following convergence error (but only when running over the larger mesh):
```
Traceback (most recent call last):
  File "/home/gecoomb/PycharmProjects/PolarRoute/venv/bin/optimise_routes", line 33, in <module>
    sys.exit(load_entry_point('polar-route', 'console_scripts', 'optimise_routes')())
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/utils.py", line 231, in wrapper
    res = func(*args, **kwargs)
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/cli.py", line 143, in optimise_routes_cli
    rp.compute_routes()
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/route_planner.py", line 514, in compute_routes
    self._dijkstra(wpt)
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/route_planner.py", line 480, in _dijkstra
    self._neighbour_cost(wpt_name,minimum_objective_index)
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/route_planner.py", line 436, in _neighbour_cost
    traveltime, crossing_points,cell_points,case = cost_func.value()
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/crossing.py", line 449, in value
    TravelTime,CrossPoints,CellPoints = self._latitude()
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/crossing.py", line 323, in _latitude
    y,TravelTime   = self._newton_optimisation(self._F,x,a,Y,Su,Sv,Nu,Nv,Ssp,Nsp)
  File "/home/gecoomb/PycharmProjects/PolarRoute/polar_route/crossing.py", line 107, in _newton_optimisation
    raise Exception('Newton not able to converge')
Exception: Newton not able to converge

```

**Comment by gecoombs** [comment]
This issue was caused by the crossing point calculation in the dijkstra route planning raising an exception when unable to converge instead of setting the relevant point as unreachable and attempting to route via an alternative path. Now fixed on the linked branch.

**Comment by gecoombs** [comment]
Fixed in PR #212
